src/cluster.h

```diff
@@ -134,4 +134,5 @@ int isNodeAvailable(clusterNode *node);
 long long getNodeReplicationOffset(clusterNode *node);
 sds aggregateClientOutputBuffer(client *c);
 void resetClusterStats(void);
+int verifyPortNumber(client *c, long long *port, long long *cport);
```

Comment:
This function does more than just verification, it also parses and assigns port values. Consider renaming it to something like `parseAndValidatePorts`  for better clarity.

Reply:
It makes sense to me. Will update.